.travis.yml

```diff
@@ -23,5 +33,5 @@ install:
   - composer install --prefer-dist
 
 script:
-  - vendor/bin/phpcs --standard=PSR2 src -p
+  - bin/phpcs --standard=phpcs.xml src
```

Comment:
@shrikeh 

Question is, do we want to use both `phpcs` and `php-cs-fixer` or will one tool be enough?